client/src/components/AlumniDirectory.js

```diff
@@ -455,7 +460,7 @@ export default class AlumniDirectory extends Component {
             case 'gradYear':
                 return (
                     <Dropdown 
-                        placeholder='Year:'
+                        placeholder={'Graduation Year'}
```

Comment:
Do the placeholders that are just a string need to be encapsulated as JSX? I think it should be fine to just leave it as `placeholder="Graduation Year"` and `placeholder="Interest"`?